src/vit_prisma/prisma_tools/loading_from_pretrained.py

```diff
@@ -13,6 +13,8 @@
 from transformers import AutoConfig, ViTForImageClassification, VivitForVideoClassification, CLIPModel
 
 import timm
+import clip
```

Comment:
It's messy but is_clip actually still loads from hugging face transformer library, we don't use the clip library. 
Based on the following code it seems like maybe all you need to do is check if AutoConfig has an attribute 'vision_config'
`    elif is_clip: # Extract vision encoder from dual-encoder CLIP model.
        hf_config = AutoConfig.from_pretrained(model_name).vision_config`

Reply:
Ok. Thanks for the tip!